dialogue-clients/build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
     implementation 'org.slf4j:slf4j-api'
 
     annotationProcessor 'org.immutables:value'
-    compile 'org.immutables:value::annotations'
+    api 'org.immutables:value::annotations'
```

Comment:
shouldn't this be compileOnly?

Reply:
EDIT nvm I just added a bunch compileOnly's and it works now :)